tests/storage/restricted_namespace_cloning/conftest.py

```
@@ -59,16 +60,64 @@ def cluster_role_for_creating_pods(admin_client):
+@pytest.fixture(scope="module")
+def dv_multi_storage_scope_module(
+    request,
+    namespace,
+    storage_class_matrix__module__,
+    fedora_ds_scope_module,
+):
+    """
+    Create a source DV in the test namespace by cloning from the golden image DataSource.
+    When cloning from a DataSource, the target DV must be at least as large as the source.
+    """
+    storage_class = [*storage_class_matrix__module__][0] if storage_class_matrix__module__ else None
```

Ruff (0.15.2)

[warning] Prefer `next(iter(storage_class_matrix__module__))` over single element slice

Replace with `next(iter(storage_class_matrix__module__))`

(RUF015)